workshops/tfx-caip-tf21/lab-01-tfx-walkthrough/lab-01.ipynb

```diff
@@ -71,7 +71,7 @@
     "from tfx.components import BigQueryExampleGen\n",
```

Comment:
<ul><li>"is able to able to generate" --&gt; "is able to generate"</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='82'/>

Reply:
Thanks. Updated.